### PR TITLE
Invasive App Updates

### PR DESCRIPTION
**Pull request by amichard**
Changelog:
- Moved Clean, Drain, Dry... toggle outside high risk form
- Removed adult dreissenid found section
- Added Appendix B toggle in high risk form
- Includes Firebase and Crashalytics fix from Micheal
- Enabled BCeID login
- Fixed some variable spelling mistakes
- Fixed Tests
- buildAndDeploy.sh to build and deploy to test flight

### CODE REVIEW

File: InspectUITests/shift page.swift [hunk 1]
```diff
@@ -35,15 +35,19 @@ class shift_page: XCTestCase {
     
     func login() {
         let app = XCUIApplication()
-        let loginButton = app.buttons["Login with IDIR"]
+//        let loginButton = app.buttons["Login with IDIR"]
```
Comment (micheal-w-wells) [inline InspectUITests/shift page.swift:38]: I think you were going to undo this?

File: ipad.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2011,10 +1990,10 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.7;
+				MARKETING_VERSION = 2.1;
 				PRODUCT_BUNDLE_IDENTIFIER = ca.bc.gov.InvasivesBC;
 				PRODUCT_NAME = Inspect;
-				PROVISIONING_PROFILE_SPECIFIER = "";
+				PROVISIONING_PROFILE_SPECIFIER = "InvasivesBC Muscles";
```
Comment (micheal-w-wells) [inline ipad.xcodeproj/project.pbxproj:1996]: Just confirming that this is not a typo on our part - doesn't really matter if that's how it's spelled as long as it's working.

File: ipad/ViewControllers/Login/LoginViewController.swift [hunk 3]
```diff
@@ -80,6 +80,12 @@ class LoginViewController: BaseViewController {
     
     // MARK: Style
     private func style() {
+        if APIURL.baseURL == "https://api-dev-invasivesbc.apps.silver.devops.gov.bc.ca/api" ||
+            APIURL.baseURL == "https://api-test-invasivesbc.apps.silver.devops.gov.bc.ca/api" {
+            loginWithBCeIDButton.isHidden = false
```
Comment (micheal-w-wells) [inline ipad/ViewControllers/Login/LoginViewController.swift:85]: Again just pointing out BCEID stuff - might need to clean this up / make sure it still works for IDIR.

**Review by micheal-w-wells — COMMENTED** [review]
Hi Richard!  All looks great but I think there's some temporary BCEID / IDIR stuff that might need to be changed, or at least validated that it still works for IDIR.

File: ipad.xcodeproj/project.pbxproj (hunk 2 above)
Comment (amichard) [reply ipad.xcodeproj/project.pbxproj:1996]: ya. unfortunately, i think thats how the app was listed, so the profile that was issued to us was the same.

File: InspectUITests/shift page.swift (hunk 1 above)
Comment (amichard) [reply InspectUITests/shift page.swift:38]: oh this is to get the testing working. but now that i think about it, we can get away with environment variables so we can get away with that, ill disable the bceid logins

**Review by micheal-w-wells — APPROVED** [review]

**Pull request merged**